### DISCUSSION TITLE
scv.pl.velocity plot blurry

### DISCUSSION THREAD

**Discussion by ccshao** [theislab/scvelo#470] [Q&A]
I tried a few combinations of figsize (up to (70, 50)) and dpi (from 150 to 1200), but the dots in the spliced/unspliced expression, velocity plot and expression are blurry. The generated files are quite small, less than 1mb for a single cell. Both png and pdf don't work well. 

I am using scvelo 0.2.2  under python 3.8.5.

**Comment by WeilerP** [comment]
To increase the resolution of the saved figures, you can specify `dpi_saved`, e.g.

```python
scv.settings.set_figure_params('scvelo', dpi_save=400)
```

**Reply by yuhao-blsa** [reply]
Hi @WeilerP. I tried the code, but it does not work either. The pdf size is the same. Any more suggestions would be appreciated.
`scv.settings.set_figure_params('scvelo', dpi_save=400)`
`scv.pl.scatter(adata, basis=top_genes[:10], frameon=False,save = "Velocity_embedding_genes.pdf",size=10, ncols=5,dpi=400)`

**Reply by WeilerP** [reply]
@JohnGenome, could you please update to the latest scVelo version and try again?

**Reply by yuhao-blsa** [reply]
It works only if I plot one gene using v0.2.4. I'll try the latest version later.
